src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -505,18 +505,34 @@ private void startFailsafeOperationThread() {
 
   @Override
   public void start(String clientPublicName) throws IOException {
-
     // Construct a single redis client to be used throughout the entire backplane.
     // We wish to avoid various synchronous and error handling issues that could occur when using
     // multiple clients.
     client = new RedisClient(jedisClusterFactory.get());
 
+    this.prequeue = createPrequeue(config);
+    this.operationQueue = createOperationQueue(config);
```

Comment:
No need for `this.` if it doesn't shadow a parameter.

Reply:
fixed